sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/models/ImageAnalysisSkill.java

```diff
@@ -115,24 +115,24 @@ public ImageAnalysisSkill setDetails(List<ImageDetail> details) {
     }
 
     /**
-     * Set the details property: A string indicating which domain-specific details to return.
+     * Set the visualFeatures property: A list of visual features.
```

Comment:
nit: just reordering? and it seems setDetails() should go before setVisualFeatures()?

Reply:
Yes, just a re-ordering. Previously, the class was hand-written so ordering was better controlled, but now it is codegen and the generator and customizations result in a different ordering